src/foam/parsers/HTMLLexer.js

```diff
@@ -29,6 +29,23 @@ foam.ENUM({
 
 foam.CLASS({
   package: 'foam.parsers',
+  name: 'Attribute',
```

Comment:
This and Tag should probably either be called HTML(Classname) or move to foam.parsers.html